test/test_core.py

```diff
@@ -7007,14 +7005,14 @@ def test_EXPORTED_RUNTIME_METHODS(self):
     '': [],
     'minimal_runtime': ['-sMINIMAL_RUNTIME=1']
   })
+  @no_wasm2js('wasm2js incompatible with Bigint')
```

Comment:
I don't see `WASM=0` used in this test..

Reply:
When the 'wasm2js1' suite is run, all the tests are implicitly run with wasm2js, and this test fails.
Having said that, maybe the test just needs to be fixed so it works both ways?

Reply:
But this test doesn't explicitly turn on WASM_BIGINT does it?   It tests the default and then it tests `-sWASM_BIGINT=0` , both of which should work with wasm2js.

Also, there is a  `if self.is_wasm2js(): ..skip` in the middle of the test.

Reply:
Wait actually it does explicitly turn on WASM_BIGINT, in the 'args' var. And the skipTest is for MEMORY64. Are we looking at the same test?

Reply:
This comment thread is about `test_dyncall_specific`.  I only see `-sMINIMAL_RUNTIME=1` in the incoming args, and `-DWASM_BIGINT` below (which is different to `-sWASM_BIGINT`).

Reply:
Ah right. I still don't see any skipping if is_wasm2js. And the test fails when run with the wasm2js1 suite.

Reply:
Sorry I misread the diff.

Why does the test fail under `wasm2js1`?   Maybe update the comment?

Reply:
Is it something like "test depends on WASM_BIGINT which is not enabled under wasm2js"?

Reply:
Ah ok the test just has 2 different flavors, one for bigint and one for without. I forgot that I removed the conditional (If get_settings(WASM_BIGINT)). I think before we just might have never been running the bigint version, since I don't think any of our regular suites turned on bigint before, right? I think the right thing to do is in the wasm2js test runner where we turn on wasm2js, also turn off bigint explicitly, and just put the conditional back here, but inverted.

Reply:
ah ok ,and what's happening here is that for some reason `self.get_setting('WASM_BIGINT')` is returning false. not sure why.

Reply:
Does that just come from settings.js? or does the test runner get it somewhere else?

Reply:
No `self.get_setting` is a map of explicitly set setting only.. its basically querying what will become the command line.

Reply:
Yeah I fixed it to check that and also the value.